custom/ilsgateway/tests/handlers/arrived.py

```diff
@@ -1,38 +0,0 @@
-from custom.ilsgateway.tanzania.reminders import ARRIVED_HELP, ARRIVED_DEFAULT, ARRIVED_KNOWN
-from custom.ilsgateway.tests.handlers.utils import ILSTestScript
-
-
-class ILSArrivedTest(ILSTestScript):
```

Comment:
they're all consolidated in `test_handlers` i believe. github is just not showing the diff